coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
Anyways, we'll need to change `coala/coala-bears`.
Ref. https://github.com/coala/coala/pull/4320#issuecomment-307619046
May be as part of that, we can also remove LOG_LEVEL from there.

But again, we'll have to see if any other project also uses it.